src/CommunityToolkit.Maui/Behaviors/ProgressBarAnimationBehavior.shared.cs

```diff
@@ -37,7 +37,7 @@ public class ProgressBarAnimationBehavior : BaseBehavior<ProgressBar>
 	}
 
 	/// <summary>
-	/// Value of <see cref="ProgressBar.Progress"/>, clamped to a minimum value of 0 and a maximum value of 1
+	/// CurrentRating of <see cref="ProgressBar.Progress"/>, clamped to a minimum value of 0 and a maximum value of 1
```

Comment:
I feel this replacement was an accident... Can you double check this and others?

Reply:
Yeah it's an accident. Sorry. i will remove it.